vowpalwabbit/parse_args.cc

```diff
@@ -1299,8 +1302,13 @@ void parse_reductions(options_i& options, vw& all)
 {
   std::vector<reduction_setup_fn> reductions;
 
+  bool python_base_reduction = false;
+
   // Base algorithms
-  reductions.push_back(GD::setup);
+  if (python_base_reduction)
```

Comment:
GD is not the only base learner, any in this first block can act as a base learner.